### ISSUE TITLE
About the loss.backward() error reporting

### ISSUE THREAD

**Issue by Shaoyi0520** [traveller59/spconv#465]
Hi, I'm having some trouble with spconv2.x. I set a small batch_size and use nvidia-smi to query the GPU still has a lot of free video memory available, but when I increase the batch, the loss.backward() gives an error, I don't understand why this is? Can you point out some possible reasons for this?
![image](https://user-images.githubusercontent.com/60254694/162701776-f9b22f39-71ad-4347-a2b0-ef205c253707.png)
